### PR TITLE
Mirror Chrome to Opera + Chrome/Samsung Internet/WebView Android

### PR DESCRIPTION
**Pull request by queengooborg**
This PR mirrors the data from Chrome Desktop to the other Chrome derivatives (particularly Chrome Android, Samsung Internet, and WebView Android), as well as some Opera data.  This PR focuses solely on changes that are straightforward and easy to review.

### CODE REVIEW

File: api/GamepadButton.json [hunk 1]
```diff
@@ -84,9 +98,16 @@
...
             ],
-            "chrome_android": {
-              "version_added": true
-            },
+            "chrome_android": [
+              {
+                "version_added": "35"
+              },
+              {
+                "prefix": "webkit",
```
Comment (foolip) [inline api/GamepadButton.json:106]: I don't trust the data being mirrored here. It's not very likely that both the `GamepadButton` interface and the `pressed` property were webkit-prefixed. Most likely the prefix data should be removed from all sub-features here.

Backing out these changes and doing it separately would be best.

File: api/GamepadEvent.json [hunk 2]
```diff
@@ -189,9 +196,16 @@
...
             },
-            "samsunginternet_android": {
-              "version_added": true
-            },
+            "samsunginternet_android": [
+              {
+                "version_added": "3.0"
+              },
+              {
+                "prefix": "webkit",
```
Comment (foolip) [inline api/GamepadEvent.json:204]: Same here, I don't trust the data being mirrored, this `gamepad` property probably wasn't prefixed.

File: api/HTMLMediaElement.json [hunk 3]
```diff
@@ -2209,7 +2212,7 @@
               "version_added": "43"
             },
             "chrome_android": {
-              "version_added": true
+              "version_added": "43"
```
Comment (foolip) [inline api/HTMLMediaElement.json:2215]: This copying incorrect data, a case of https://github.com/mdn/browser-compat-data/issues/7843. Can back out these changes you fix all of the "43" values in this file, updating both Chrome and mirrored data at the same time?

File: api/IDBKeyRange.json [hunk 4]
```diff
@@ -50,12 +57,26 @@
...
-            "version_added": true
-          }
+          "samsunginternet_android": [
+            {
+              "version_added": "1.5"
+            },
+            {
+              "version_added": "1.5",
+              "version_removed": "7.0",
+              "prefix": "webkit"
```
Comment (foolip) [inline api/IDBKeyRange.json:67]: This prefixed support should be on Opera as well. For consistency, can you add it there (my preference) or alternatively skip it here too?

File: api/IDBVersionChangeEvent.json [hunk 5]
```diff
@@ -49,9 +55,15 @@
...
-          "samsunginternet_android": {
-            "version_added": true
-          },
+          "samsunginternet_android": [
+            {
+              "version_added": "1.5"
+            },
+            {
+              "version_added": "1.0",
+              "prefix": "webkit"
```
Comment (foolip) [inline api/IDBVersionChangeEvent.json:64]: Can you add this for Opera too?

File: api/Metadata.json [hunk 6]
```diff
@@ -38,7 +38,7 @@
             "version_added": false
           },
           "samsunginternet_android": {
-            "version_added": true,
+            "version_added": "1.0",
             "prefix": "webkit"
```
Comment (foolip) [inline api/Metadata.json:42]: This is missing from Opera, but in this case the Opera data is false, so wrong in a more easily detectable way. It's fine to leave that wrong for now.

**Review by foolip — CHANGES_REQUESTED** [review]
Found some things... backing out those changes and landing the rest would be most expedient.

File: api/GamepadButton.json (hunk 1 above)
Comment (queengooborg) [reply api/GamepadButton.json]: Opened #13269 to take care of the gamepad data!

File: api/HTMLMediaElement.json (hunk 3 above)
Comment (queengooborg) [reply api/HTMLMediaElement.json]: This is really odd, running the collector on Chrome 1 shows support all the way back then, but the collector isn't updating this data...  I'll make a note of this in the related collector's issue.

**Review by foolip — APPROVED** [review]
I've used https://github.com/mdn/browser-compat-data/pull/6862 to look at the results in a more aggregated way and spot checked some things without finding any additional issues.

No doubt some of the data being mirrored here is correct, so if anyone finds this PR in the future, don't trust it any more than you would the Chrome data being mirrored.

**Pull request merged**